js/views/attachment_view.js

```diff
@@ -198,7 +208,15 @@
         if (!this.objectUrl) {
             this.objectUrl = window.URL.createObjectURL(this.blob);
         }
-        this.view = new View(this.objectUrl, this.model.contentType);
+
+        if (this.isImage()) {
+          window.autoOrientImage(this.objectUrl).then(dataURL => {
+            this.autoOrientedImageDataURL = dataURL;
+          });
```

Comment:
Since the rest of the code path for `LightboxView` is synchronous, I start the async auto-orientation process here and `LightboxView` can use it if ready, otherwise it’ll fall back on `objectUrl`. Will think about this some more to see if there is a cleaner way.